docs/api/web-contents.md

```diff
@@ -412,6 +426,39 @@ Executes the editing command `replace` in web page.
 
 Executes the editing command `replaceMisspelling` in web page.
 
+### `webContents.findInPage(id, text[, options])`
+
+* `id` Integer
+* `text` String - Content to be searched, must not be empty.
+* `options` Object __Optional__
+  * `forward` Boolean - Whether to search forward or backward.
+  * `findNext` Boolean - Whether the operation is first request or a follow up.
+  * `matchCase` Boolean - Whether search should be case-sensitive.
+  * `wordStart` Boolean - Whether to look only at the start of words.
+  * `medialCapitalAsWordStart` Boolean - When combined with `wordStart`,
+    accepts a match in the middle of a word if the match begins with an
+    uppercase letter followed by a lowercase or non-letter.
+    Accepts several other intra-word matches
+
+Finds all matches for the `text` in the web page.
+
+### `webContents.stopFindInPage(action)`
+
+* `action` String - Should be called with either `clearSelection` or `keepSelection`.
+                    By default it keeps the last selection.
```

Comment:
The `action` doesn't have a default value so "By default" is very confusing. We should add an explicit option for `STOP_FIND_ACTION_ACTIVATE_SELECTION`, and document what does each `action` do.

Here are some comments on what they do:
https://code.google.com/p/chromium/codesearch#chromium/src/chrome/browser/ui/find_bar/find_bar_controller.h&sq=package:chromium&type=cs&l=27